### PR TITLE
[BE] Add optional step to upload coredumps

### PR DESCRIPTION
**Pull request by malfet**
Test plan: cause a crash and make sure core is uploaded

### CODE REVIEW

File: .github/templates/common.yml.j2 [hunk 1]
```diff
@@ -309,6 +309,25 @@ concurrency:
...
+        with:
+{%- if config != "" and shard != "" %}
+          name: !{{ artifact_name }}-!{{ config }}-!{{ shard }}
+{%- else %}
+          name: !{{ artifact_name }}
+{%- endif %}
+          retention-days: 14
+          if-no-files-found: ignore
+          path:
+            **/core.[1-9]*
```
Comment (janeyx99) [inline .github/templates/common.yml.j2:328]: shellcheck doesn't like starting with a * this for some reason

**Review by janeyx99 — COMMENTED** [review]
LGTM once lint is fixed :)

File: .github/templates/common.yml.j2 (hunk 1 above)
Comment (malfet) [reply .github/templates/common.yml.j2]: Yeah, added `./**/core.[1-9]*`

**Review by janeyx99 — APPROVED** [review]
forgot to click accept last time oops

**Pull request closed without merging**